Week-6/exercise-6.6/closeSum.js

```diff
@@ -0,0 +1,7 @@
+// Given an array S of n integers,
+// find three integers in S such that the sum is closest to a given number, target.
+// Return the sum of the three integers.
+// Assume that there will only be one solution
+
+s = [-1, 2, 1, -4];
+target = 1;
```

Comment:
1. Sort the given array.
2. Loop over the array and fix the first element of the possible triplet, arr[i].
3. Then fix two pointers, one at (i + 1) and the other at n – 1. And look at the sum,
    1. If the sum is smaller than the sum we need to get to, we increase the first pointer.
    2. Else, If the sum is bigger, Decrease the end pointer to reduce the sum.
    3. Update the closest sum found so far.